src/api/app/views/webui2/webui/user/_info.html.haml

```diff
@@ -0,0 +1,63 @@
+= webui2_user_image_tag(user, 'home-avatar')
+
+%h4.home-usertitle
+  .home-username
+    = user.realname
+  .home-login
+    = user.login
+
+- unless user.is_nobody?
+  %p
+    = mail_to user.email, title: "Send mail to #{user.name}" do
+      %i.fas.fa-envelope
+      = user.email
+
+- if groups.any?
+  Member of the #{'group'.pluralize(groups.size)}
+  %ul
+    - groups.each do |group|
+      %li= link_to("#{group.title} (#{group.tasks})", group_show_path(group))
+
+- if role_titles.any?
+  Has the #{'role'.pluralize(role_titles.size)}
+  %ul
+    - role_titles.each do |title|
+      %li= title
+
+- if is_user
+  - if user.in_beta?
```

Comment:
I have moved this. Before everybody could see if a user is in the beta program. I think that only the user should see this. Maybe we should change it in the old UI as well.